projects/core/build/tokens.ts

```diff
@@ -82,6 +82,7 @@ const color = {
   },
   blue: {
     50: token([198, 100, 95]),
+    75: token([198, 100, 94]),
```

Comment:
We need an updated design for https://v16-beta.clarity.design/documentation/color. Adding blue-75 will throw off the other columns.

Reply:
@kevinbuhmann Good catch. Can you file a ticket in JIRA to track that and assign it to the appropriate person?

Reply:
CDE-1187

Reply:
This is correct.